.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "rust-vmm-ci"]
```

Comment:
You're missing the `coverage_config.json` file.

Reply:
fixed